sdk/src/main/java/com/hedera/hashgraph/sdk/EthereumTransactionDataLegacy.java

```diff
@@ -0,0 +1,83 @@
+package com.hedera.hashgraph.sdk;
+
+import com.esaulpaugh.headlong.rlp.RLPDecoder;
+import com.esaulpaugh.headlong.rlp.RLPEncoder;
+import com.esaulpaugh.headlong.rlp.RLPItem;
+import com.google.common.base.MoreObjects;
+import org.bouncycastle.util.encoders.Hex;
+
+import java.math.BigInteger;
+import java.util.List;
+
+public class EthereumTransactionDataLegacy extends EthereumTransactionData {
+    public byte[] nonce;
+    public byte[] gasPrice;
+    public byte[] gasLimit;
+    public byte[] to;
+    public byte[] value;
+    public byte[] v;
+    public byte[] r;
+    public byte[] s;
+
+    EthereumTransactionDataLegacy(
+        byte[] nonce,
+        byte[] gasPrice,
+        byte[] gasLimit,
+        byte[] to,
+        byte[] value,
+        byte[] callData,
+        byte[] v,
+        byte[] r,
+        byte[] s
+    ) {
+        super(callData);
+
+        this.nonce = nonce;
+        this.gasPrice = gasPrice;
+        this.gasLimit = gasLimit;
+        this.to = to;
+        this.value = value;
+        this.v = v;
```

Comment:
note: I believe recoveryId may be present and can be extracted from v
```java
        var v = rlpItems.get(6).asBytes();
        BigInteger vBi = new BigInteger(1, v);
        this.recId = vBi.testBit(0) ? 0 : 1
```

Reply:
Where would it be present? I went off https://github.com/hashgraph/hedera-services/blob/ac677f00931a672d077069cbd44a8828d54017a5/hapi-utils/src/main/java/com/hedera/services/ethereum/EthTxData.java#L98 which doesn't seem to ever check for a potential `recoveryId`.

Reply:
Pointing it out in case you want to include both `recoveryId` and `chainId` in your `toString()` like the `EthTxData.java` example you provided does.
If not then this is a no-op

Reply:
I seem to have misunderstood what you meant, I went ahead and added `recoveryId` and `chainId` to `*Legacy`.